Physicist/Physicist/Controls/SoundController.cs

```diff
@@ -0,0 +1,191 @@
+        {
+            SoundEffect effect = ContentController.Instance.GetContent<SoundEffect>(name);
+            SoundEffectInstance instance = effect.CreateInstance();
+            
+            instance.Volume = volume;
+            instance.Pitch = pitch;
+            instance.Pan = pan;
+            instance.IsLooped = looping;
+
+            if (emitter != null)
+            {
+                instance.Apply3D(SoundController.listener, emitter);
+            }
+
+            if (SoundController.instances == null)
+            {
+                SoundController.instances = new Dictionary<Guid, SoundEffectInstance>();
+            }
+
+            Guid id = new Guid();
+            SoundController.instances.Add(id, instance);
+            instance.Play();
+            return id;
+        }
+
+        public static void StopSound(Guid id)
+        {
+            if (SoundController.instances.ContainsKey(id))
+            {
+                SoundEffectInstance instance = SoundController.instances[id];
+                if (instance.State == SoundState.Playing)
+                {
+                    instance.Stop();
+                    instance.Dispose();
+                    SoundController.instances.Remove(id);
+                }
+            }
+        }
+
+        public static void PauseSound(Guid id)
+        {
+            if (SoundController.instances.ContainsKey(id))
+            {
+                SoundEffectInstance instance = SoundController.instances[id];
+                if (instance.State == SoundState.Playing)
+                {
+                    instance.Pause();
+                }
+            }
+        }
+
+        public static void ResumeSound(Guid id)
+        {
+            if (SoundController.instances.ContainsKey(id))
+            {
+                SoundEffectInstance instance = SoundController.instances[id];
+                if (instance.State == SoundState.Paused)
+                {
+                    instance.Resume();
+                }
+            }
+        }
+
+        public static bool SoundIsPlaying(Guid id)
+        {
+            if (SoundController.instances.ContainsKey(id))
+            {
+                SoundEffectInstance instance = SoundController.instances[id];
+                return instance.State == SoundState.Playing;
+            }
+
+            return false;
+        }
+
+        public static void Update()
+        {
+            for (int i = SoundController.instances.Count - 1; i >= 0; i--)
+            {
+                var entry = SoundController.instances.ElementAt(i);
+                if (entry.Value.State == SoundState.Stopped)
+                {
+                    entry.Value.Dispose();
+                    SoundController.instances.Remove(entry.Key);
+                }
+            }
+
+            if (Map.Players.Count() > 0 && Map.Players.ElementAt(0) != null)
```

Comment:
Instead of trying to dynamically set this once, then forgetting it. Expose the listener and set it in PhysicistGameScreen after the map has loaded. See above in PhysicistGameScreen.cs